packages/dashboard-core-plugins/src/panels/IrisGridPanelTooltip.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, useEffect, useState } from 'react';
```

Comment:
`useEffect`, `useState` unused
```suggestion
import React, { ReactElement } from 'react';
```